pages/DSNP/Identity.md

```diff
@@ -19,13 +19,27 @@ A user's ownership of their identity is expressed via ownership and control of t
 Control entails the power to announce content associated with the identifier and
 the ability to delegate permission to others to announce content on the user's behalf.
 
+### Retirement
+
+A user may choose to retire their identity at any time.
+Once an identifier is retired, an implementation MAY remove all state data associated with that identifier, provided that an indication that the identifier is retired remains, so it may not be reused in the future.
```

Comment:
Do we want to use an upper-case I on Identifiers as we use an upper-case A on Announcements?  The capitalization of Identifiers is not currently completely consistent within the document

Reply:
I like that suggestion. I'll change all usage in that document.